app/test/test_cruds.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+import os
+import json
+from v1 import create_app
+
+class booksTestCase(unittest.TestCase):
+    """In this class we shall be testing the API endpoints"""
+
+    def setUp(self):
+        """Define test variables and initialize some list data for the app."""
+        self.app = create_app(config_name="testing")
+        self.client = self.app.test_client
+        self.book = {'ISBN': '475', 'title':'Night train to Istanbull','author':'Penny Mice',
+                      'edition':'1','publisher':'Pearson','copies':'54'}
+
```

Comment:
Why have two lines between functions?

Reply:
Was trying to avoid an extremely long line of code...